### PR TITLE
Fixes a crash due to a runtime error on < iOS 11

### LINKED ISSUE #578: App crashes while selecting a notification
If I try to select a notification by tapping it on the overview screen. The app crashes.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

### PR DESCRIPTION
**Pull request by BasThomas**
Fixes #578
ObjC 😕

### CODE REVIEW

File: Local Pods/SlackTextViewController/Source/SLKTextViewController.m [hunk 1]
```diff
@@ -2260,16 +2260,19 @@ - (void)slk_setupViewConstraints
...
-    trailingSafe.constant = 0;
-    [NSLayoutConstraint activateConstraints:@[leadingSafe, trailingSafe]];
+    if (@available(iOS 11, *)) {
+        NSLayoutConstraint *leadingSafe = [self.scrollViewProxy.leadingAnchor constraintEqualToSystemSpacingAfterAnchor:self.view.safeAreaLayoutGuide.leadingAnchor multiplier:1.0];
+        leadingSafe.constant = 0;
+        NSLayoutConstraint *trailingSafe = [self.scrollViewProxy.trailingAnchor constraintEqualToSystemSpacingAfterAnchor:self.view.safeAreaLayoutGuide.trailingAnchor multiplier:1.0];
+        trailingSafe.constant = 0;
+        [NSLayoutConstraint activateConstraints:@[leadingSafe, trailingSafe]];
+    } else {
+        [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|[scrollView]|" options:0 metrics:nil views:views]];
```
Comment (BasThomas) [inline Local Pods/SlackTextViewController/Source/SLKTextViewController.m]: Not sure if you removed this one for another reason? Should we drop it completely?

**Comment by weyert** [comment]
Thanks @BasThomas! I can't upgrade my phone to iOS 11 as the radio app on my phone is still 32 bits 😅

**Comment by rnystrom** [comment]
Oops duh, thanks!

**Comment by rnystrom** [comment]
We can actually ignore this since I did this fix totally wrong. Doing it the "right" way cell-level insetting using layout guides. See #581.

**Pull request closed without merging**